autorag/deploy.py

```diff
@@ -0,0 +1,134 @@
+import logging
+import os
+import uuid
+from typing import Optional, Dict
+
+import pandas as pd
+import yaml
+
+from autorag.schema.module import SUPPORT_MODULES
+from autorag.utils.util import load_summary_file
+
+logger = logging.getLogger("AutoRAG")
+
+
+def summary_df_to_yaml(summary_df: pd.DataFrame) -> Dict:
+    """
+    Convert trial summary dataframe to config yaml file.
+
+    :param summary_df:
+    :return: Dictionary of config yaml file.
+        You can save this dictionary to yaml file.
+    """
+    # summary_df columns : 'node_line_name', 'node_type', 'best_module_filename',
+    #                      'best_module_name', 'best_module_params', 'best_execution_time'
+    grouped = summary_df.groupby('node_line_name')
+
+    node_lines = [
+        {
+            'node_line_name': node_line_name,
+            'nodes': [
+                {
+                    'node_type': row['node_type'],
+                    'modules': [{
+                        'module_type': row['best_module_name'],
+                        **row['best_module_params']
+                    }]
+                }
+                for _, row in node_line.iterrows()
+            ]
+        }
+        for node_line_name, node_line in grouped
+    ]
+    return {'node_lines': node_lines}
+
+
+def extract_pipeline(trial_path: str, output_path: Optional[str] = None) -> Dict:
```

Comment:
In my think, exact_pipeline is not good for function name. 

function name end with pipeline, In framework like AutoRAG, llamaindex, make me feel module is pipeline module.
(maybe?)
umm... little worried but, I don't have clear idea about this feeling